tools/tesseratos/src/tesseratos/console/plugin.cpp

```diff
@@ -0,0 +1,290 @@
+#include <imgui.h>
+
+#include <cubos/core/log.hpp>
+
+#include <cubos/engine/imgui/plugin.hpp>
+
+#include <tesseratos/console/plugin.hpp>
+#include <tesseratos/toolbox/plugin.hpp>
+
+using cubos::engine::Cubos;
+
+using namespace tesseratos;
+
+namespace
+{
+    struct Mode
```

Comment:
```suggestion
    struct State
```
for consistency with other plugins